Zadatak1.ps1

```diff
@@ -0,0 +1,59 @@
+﻿param(
+    [Parameter(Mandatory=$true)]
```

Comment:
The parameter should not be mandatory. Script should create statements for all the available dates if `dateFolder` is not given.